File: TS_JT-Reader/TS_JT-Reader_Vis-Studio/TS_JT-Reader/ArithmeticProbabilityRange.ts
module C_sharp_JT_Reader.Codecs {
    export class ArithmeticProbabilityRange {
        private m_low: number;
        private m_high: number;
        private m_scale: number;
        constructor(low: number, high: number, scale: number) {
            this.m_low = low;
            this.m_high = high;
            this.m_scale = scale;
        }
        public getLow(): number {
            return this.m_low;
        }
        public getHigh(): number {
            return this.m_high;
        }
        public getScale(): number {
            return this.m_scale;
        }
    }
}

File: TS_JT-Reader/C#_JT-Reader_Vis-Studio/C sharp JT-Reader/Codecs/ArithmeticProbabilityRange.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace C_sharp_JT_Reader.Codecs
{
    public class ArithmeticProbabilityRange
    {
        private long m_low;
        private long m_high;
        private long m_scale;

        public ArithmeticProbabilityRange(long low, long high, long scale)
        {
            this.m_low = low;
            this.m_high = high;
            this.m_scale = scale;
        }

        public long getLow()
        {
            return this.m_low;
        }

        public long getHigh()
        {
            return this.m_high;
        }

        public long getScale()
        {
            return this.m_scale;
        }
    }
}
